lib/kubernetes-deploy/options_helper.rb

```diff
@@ -29,6 +29,14 @@ def with_processed_template_paths(template_paths, require_explicit_path: false)
         end
       end
 
+      def with_processed_task_template(template_path)
```

Comment:
`OptionsHelper.with_processed_template_paths` both manipulates **and** validates the value(s) we pass to it, so its block interface makes sense.

In this particular case we're just validating the value so maybe we could change the name to reflect that. What do you think?

Also, we maybe don't need the block interface, but I have no strong feelings about that.

Reply:
Thanks for the comment @dirceu , I remove the check `OptionsHelper.with_processed_template_paths` as it should be backwards compatible: https://github.com/Shopify/kubernetes-deploy/pull/579#discussion_r332127566